### ISSUE TITLE
Service template cannot be instantiated

### ISSUE THREAD

**Issue by Yenya** [OpenNebula/one#5958]
**Description**
Service template with a mandatory custom variable with empty description cannot be instantiated.

**To Reproduce**
Create the service template, and in the Custom variables section add a variable (say "MYTESTVAR") of type text, empty description, empty default value, and set it as mandatory.

Try to instantiate this service template.

**Expected behavior**
There should be an input field in the "Custom variables" section during instantiation. But the section is empty instead, no input field at all and indeed when I click on Instantiate, Sunstone complains that

`Every custom_attrs key must have its value defined at custom_attrs_value`

When edit the service template again, and fill in also a Description of that variable, the input field appears in the Instantiate screen as expected, the mandatory variable can be filled in, and the service template can be instantiated.

So either the Description field should be made mandatory when editing the service template, or in the Instantiate screen Sunstone should be able to present the input fields even for variables without Description.

**Details**
 - Affected Component: Sunstone
 - Hypervisor: KVM
 - Version: 6.4.0 CE

## Progress Status
- [ ] Code committed
- [ ] Testing - QA
- [ ] Documentation (Release notes - resolved issues, compatibility, known issues)

**Comment by Yenya** [comment]
@vickmp - here is the template:

```


{
  "name": "Testservice",
  "deployment": "straight",
  "description": "",
  "roles": [
    {
      "name": "testrole",
      "cardinality": 1,
      "vm_template": 1205,
      "elasticity_policies": [],
      "scheduled_policies": [],
      "vm_template_contents": ""
    }
  ],
  "ready_status_gate": false,
  "automatic_deletion": false,
  "custom_attrs": {
    "TESTVAR": "M|text|| |"
  },
  "registration_time": 1672673539
}
```

FWIW, I also made a screen capture video of Sunstone, it is available here:

https://www.fi.muni.cz/~kas/tmp/opennebula-service-custom-var.mp4

**Comment by vickmp** [comment]
Hi @Yenya, thanks for your update. We are going to proceed to close this issue as after further analysis we have come to the conclusion that this bug was fixed in 6.4.2 through this [issue](https://github.com/OpenNebula/one/issues/5918).
